test/fixtures/musical-fixture.js

```diff
@@ -0,0 +1,249 @@
+const dataSource = require("../../models/appDataSource");
+
+const createTheaters = (theaterList) => {
+  let data = [];
+
+  for (const theater of theaterList) {
+    data.push([theater.id, theater.name, theater.totalSeats, theater.cityId, theater.thumbnailImageUrl]);
+  }
```

Comment:
- 가독성을 위해서 전체적으로 map 함수로 변경해보는 것은 어떨까요?
- 아래 빈 배열 선언 및 for 문을 돌리는 모든 곳에 전반적으로 적용해보세요!

```suggestion
 const data = theaterList.map(theater => [
     theater.id, 
     theater.name, 
     theater.totalSeats, 
     theater.cityId, 
     theater.thumbnailImageUrl
  ]);

```